seasonal/README.md

```diff
@@ -0,0 +1,64 @@
+### Seasonal branding
+
+Each folder in this directory contains branding inspired by a specific season. Such assets are temporarily applied to the server to celebrate holidays, promote on-going events, or to show support for various world-wide movements.
+
+While the primary function is to organize the various assets we have, they are also used by **Seasonalbot's branding manager**. The bot will automatically apply seasonal assets pulled directly from this repository to the server. In order for Seasonalbot to properly detect and apply assets, certain conventions must be followed.
+
+#### Automatically managed assets
+
+There are 3 types of seasonal assets: **server icons**, **server banners**, and **bot avatars**. While a season always has **at most 1** banner and bot avatar, there may be **multiple** server icons. In such a case, Seasonalbot will periodically cycle available icons in a configurable interval (once every *n* days).
+
+In order for banners and avatars to be discovered, they shall be placed directly in a seasonal directory, named `seasonal/<season_name>/banner.png` and `../avatar.png` respectively. Note that these assets are expected to *always* carry the **png** extension. Server icons shall always (regardless of whether the is only 1, or many) be placed in a nested directory, as follows: `../server_icons/festive_256.gif`. Server icons are name and extension agnostic - they are registered simply be being present in the `server_icons/` directory.
```

Comment:
Use three dots for paths to make it clearer it's an ellipses rather than "go up one directory". Also correct "be being" to "by being".
```suggestion
In order for banners and avatars to be discovered, they shall be placed directly in a seasonal directory, named `seasonal/<season_name>/banner.png` and `.../avatar.png` respectively. Note that these assets are expected to *always* carry the **png** extension. Server icons shall always (regardless of whether the is only 1, or many) be placed in a nested directory, as follows: `.../server_icons/festive_256.gif`. Server icons are name and extension agnostic - they are registered simply by being present in the `server_icons/` directory.
```